tox.ini

```diff
@@ -103,7 +103,6 @@ commands =
 [testenv:linters-py{38,39,310,311}]
 description = Run linters
 commands =
-    flake8 .{/}src
```

Comment:
I think this is what triggered `flake8`  in https://github.com/GridTools/gt4py/pull/1624#issuecomment-2324035028.

Question here is whether we still need this category (at all) because, as far as I understand, the remaining `mypy`  is also called from `pre-commit`.

Reply:
I agree, we should just completely delete the `linters` task in tox, since this has been fully replaced by pre-commit.

Reply:
removed :broom: